### PR TITLE
US132656 - Add Course Offering to Ignore List from UI (front end)

### PR DESCRIPTION
**Pull request by bamcdole**
[US132656](https://rally1.rallydev.com/#/?detail=/userstory/610991031631&fdp=true)

4XA9UmaDuA

### CODE REVIEW

File: src/components/module-scheduler-ignore-list.js [hunk 1]
```diff
@@ -61,62 +91,137 @@ class ModuleSchedulerIgnoreList extends BaseMixin(LocalizeMixin(LitElement)) {
...
 				icon="tier3:chevron-left"
 				text=${ this.localize('button:back') }>
 			</d2l-button-subtle>
 			<h2 class="d2l-ignore-list-title">${this.localize('ignoreList:title', { scheduleName: this.scheduleName })}</h2>
 			<p>${this.localize('page:description')}</p>
+			<d2l-button-subtle
+				class="d2l-add-to-ignore-list-btn"
+				@click="${this._openAddToIgnoreListDialog}"
+				text=${ this.localize('ignoreList:addToIgnoreListButton') }>
+			</d2l-button-subtle>
```
Comment (jguann) [inline src/components/module-scheduler-ignore-list.js:129]: The button in the design mock-up seems to be a primary button with a plus icon instead: https://rally1.rallydev.com/#/431372673576d/dashboard?detail=%2Fuserstory%2F610427935179&fdp=true

**Comment by bamcdole** [comment]
image

**Review by jguann — APPROVED** [review]

**Comment by bamcdole** [comment]
Seems like SauceLabs is unhappy.

File: src/services/schedule-service.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import { Requests } from '../api/requests.js';
 
 export class ScheduleService {
+
+	static async addToIgnoreList(schedule, courseOfferingId) {
+		await Requests.addToIgnoreList(schedule, courseOfferingId);
+		return;
```
Comment (jorge-ramirez-arredondo) [inline src/services/schedule-service.js:7]: So, this totally doesn't matter, but isn't the `return;` here redundant? Afaik, this is the same as having no return at all. I know we have this elsewhere, but I'm not sure I see the point there either.

**Review by jorge-ramirez-arredondo — APPROVED** [review]

**Comment by bamcdole** [comment]
BSI v20.21.12-181

**Pull request merged**